shared/message/support/GlobalConfig.proto

```diff
@@ -20,8 +20,18 @@ syntax = "proto3";
 
 package message.support;
 
-// Field dimensions as defined in the Robocup rules:
+/**
+ * Field dimensions as defined in the Robocup rules.
+ */
 message GlobalConfig {
+
+    /**
+     * A number ID, assigned beginning with 0 or 1.
```

Comment:
The robots number in the team. Starts at 1. Should be the same as the robots number (nugus1, nugus2, etc)

Reply:
Awesome, made that the comment, as that sums it up nicely